### PR TITLE
[DOCS] Replaces outdated integration guides with redirects to versioned documentation

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description:
- Replace outdated integration guides with landing pages that include a link to the most recent version of the docs to support the integration in question.

## Definition of done:
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md [hunk 1]
```diff
@@ -3,64 +3,13 @@ title: Use Great Expectations in AWS Glue
...
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+AWS Glue is a serverless data integration service by Amazon.  It makes it easier to prepare, move, and integrate data from multiple sources.  It has applications for analytics, machine learning, and application development.
 
-<InProgress />
+:::info 
 
-This Guide demonstrates how to set up, initialize and run validations against your data on AWS Glue Spark Job.
-We will cover case with RuntimeDataConnector and use S3 as metadata store.
+The most recent guide for using GX with AWS Glue is for GX version 0.15.50 and can be read in [GX's versioned integration documentation for AWS Glue](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_aws_glue).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md]:
```suggestion
To use Great Expectations (GX) with AWS Glue, see [How to use GX in AWS Glue](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_aws_glue) in version 0.15.50 of the documentation.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md [hunk 2]
```diff
@@ -3,64 +3,13 @@ title: Use Great Expectations in AWS Glue
...
 
-### 0. Pre-requirements
+Guidance on AWS Glue's usage can be found on [Amazon's AWS Glue site](https://aws.amazon.com/glue/).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md]:
```suggestion
For more information about implementing and using AWS Glue, see the [AWS Glue site](https://aws.amazon.com/glue/).
```

**Review by kwcanuck — APPROVED** [review]

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md [hunk 3]
```diff
@@ -3,64 +3,13 @@ title: Use Great Expectations in AWS Glue
 description: AWS Glue
 sidebar_custom_props: { icon: 'img/integrations/aws_glue_icon.svg' }
 ---
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
-import Congratulations from '../guides/connecting_to_your_data/components/congratulations.md'
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+AWS Glue is a serverless data integration service by Amazon.  It makes it easier to prepare, move, and integrate data from multiple sources.  It has applications for analytics, machine learning, and application development.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md:8]:
```suggestion
AWS Glue is a serverless data integration service that makes it easier to prepare, move, and integrate data from multiple sources. It includes applications for analytics, machine learning, and application development.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_deepnote.md [hunk 4]
```diff
@@ -5,245 +5,12 @@ sidebar_label: "Deepnote"
 sidebar_custom_props: { icon: 'img/integrations/deepnote_icon.png' }
 ---
 
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+Deepnote is a Jupyter compatible, cloud-hosted, data science notebook that helps teams collaborate to discover, understand and share findings in their data stack. Integrating Great Expectations with Deepnote allows documentation to be hosted and notebooks to be scheduled.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_deepnote.md:8]:
```suggestion
Deepnote is a Jupyter compatible, cloud-hosted, data science notebook that helps organizations discover, understand, and share findings in their data stack. Integrate Great Expectations (GX) with Deepnote to host documentation and schedule notebooks. 
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_deepnote.md [hunk 5]
```diff
@@ -5,245 +5,12 @@ sidebar_label: "Deepnote"
...
 
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+Deepnote is a Jupyter compatible, cloud-hosted, data science notebook that helps teams collaborate to discover, understand and share findings in their data stack. Integrating Great Expectations with Deepnote allows documentation to be hosted and notebooks to be scheduled. 
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+:::info 
 
-<InProgress />
+To use Great Expectations (GX) with Deepnote, see [How to use GX in Deepnote](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_deepnote) in version 0.15.50 of the documentation.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_deepnote.md:14]:
```suggestion
To use GX with Deepnote, see [How to use GX in Deepnote](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_deepnote) in version 0.15.50 of the documentation.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md [hunk 6]
```diff
@@ -4,178 +4,14 @@ description: "Use Great Expectations with Flyte"
...
-       dataframe=to_df(
-           dataset="https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
-       )
-   )
-
-```
-In the code, we define a `schema_task` that accepts a `dataframe` argument, which is associated with the `GreatExpectationsType`. The first argument denotes the data type, which in our case is a `FlyteSchema`, and the second argument represents the Great Expectations configuration. 
-
-The `GreatExpectationsType` generates a parquet file and copies it to the `local_file_path` when inputting the tabular data. This path is then used to perform data validation.
+The power of data validation in Great Expectations can be integrated with Flyte to validate the data moving in and out of the pipeline entities you may have defined in Flyte. This helps establish stricter boundaries around your data to ensure that everything works as expected and data does not crash your pipelines unexpectedly!
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md:9]:
```suggestion
Integrate Great Expectations (GX) with Flyte to validate the data used in your Flyte pipelines and ensure that your pipelines run without interruption.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md [hunk 7]
```diff
@@ -4,142 +4,15 @@ description: "Use Great Expectations with Meltano"
 sidebar_label: "Meltano"
 sidebar_custom_props: { icon: 'img/integrations/meltano_icon.png' }
 ---
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+Meltano is an Open Source DataOps OS that's used to install and configure data applications (Great Expectations, Singer, dbt, Airflow, etc.) that your team's data platform is built on top of, all in one central repository.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md:8]:
```suggestion
Meltano is an Open Source DataOps OS that you can use to install and configure data applications such as Great Expectations (GX) Singer, dbt, and Airflow with your existing data platform in a central repository.
```
image

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md [hunk 8]
```diff
@@ -4,142 +4,15 @@ description: "Use Great Expectations with Meltano"
...
 
-<InProgress />
+Using Meltano enables teams to easily implement DataOps best practices like configuration as code, code reviews, isolated test environments, CI/CD, etc.  A common use case is to manage ELT pipelines with Meltano and as part of ensuring the quality of the data in those pipelines, teams bring in Great Expectations.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md:10]:
```suggestion
Meltano can help you implement DataOps best practices such as configuration as code, code reviews, isolated test environments, and CI/CD.  Typically, Metano is used to manage ELT pipelines and to help ensure data quality in pipelines.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md [hunk 9]
```diff
@@ -5,224 +5,12 @@ sidebar_label: "YData-Synthetic"
 sidebar_custom_props: { icon: 'img/integrations/ydata_synthetic_icon.png' }
 ---
 
-import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+YData-Synthetic is an open-source synthetic data engine. Using different kinds of Generative Adversarial Networks (GANS), the engine learns patterns and statistical properties of original data. It can create endless samples of synthetic data that resemble the original data.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md:8]:
```suggestion
YData-Synthetic is an open-source synthetic data engine. It uses Generative Adversarial Networks (GANS) to identify patterns and statistical properties in the originating data to help it create synthetic data samples that resemble the originating data.
```

File: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md [hunk 10]
```diff
@@ -5,224 +5,12 @@ sidebar_label: "YData-Synthetic"
...
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+:::info 
 
-<InProgress />
+To use Great Expectations (GX) with YData-Synthetic, see [How to use GX with YData-Synthetic](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic) in version 0.15.50 of the documentation.
 
-_This piece of documentation was authored by [Arunn Thevapalan](https://www.linkedin.com/in/arunn-thevapalan/)._
+For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md]:
```suggestion
For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).
```
The previous instance was YData. Confirm the correct format and use it consistently.

File: docs/docusaurus/docs/integrations/integration_datahub.md [hunk 11]
```diff
@@ -8,106 +8,14 @@ sidebar_label: "DataHub"
 sidebar_custom_props: { icon: 'img/integrations/datahub_icon.png' }
 ---
 
-:::info
-* Maintained By: DataHub
-* Status: Beta
-* Support/Contact: https://slack.datahubproject.io/
-:::
+DataHub is a modern data catalog built to enable end-to-end data discovery, observability, and governance.  DataHub is built on an extensible metadata platform that helps manage the complexity of diverse and evolving data ecosystems.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/integrations/integration_datahub.md:11]:
```suggestion
DataHub is a modern data catalog built to enable end-to-end data discovery, observability, and governance.  DataHub is built on an extensible metadata platform that helps manage the complexity of diverse and evolving data ecosystems.
```

Just an aside - I'm not sure why we're explaining what the applications are and why they are helpful. I would suggest that all that is required is a link to the source documentation. Your call.

File: docs/docusaurus/docs/integrations/integration_datahub.md [hunk 12]
```diff
@@ -8,106 +8,14 @@ sidebar_label: "DataHub"
...
 
-This integration allows you to push the results of running Expectations into DataHub (https://datahubproject.io/). DataHub is a metadata platform which enables search & discovery, federated governance, and data observability for the Modern Data Stack.
 
+:::info 
 
-### Technical background
-There is a custom Action named `DataHubValidationAction` which allows you to view Expectation Results inside of DataHub.
+Consult [DataHub's metadata ingestion documentation for GX](https://datahubproject.io/docs/metadata-ingestion/integration_docs/great-expectations)
```
Comment (kwcanuck) [inline docs/docusaurus/docs/integrations/integration_datahub.md:16]:
```suggestion
See [DataHub's metadata ingestion documentation for GX](https://datahubproject.io/docs/metadata-ingestion/integration_docs/great-expectations)
```

File: docs/docusaurus/docs/integrations/integration_datahub.md [hunk 13]
```diff
@@ -8,106 +8,14 @@ sidebar_label: "DataHub"
...
+for more information on how to set up and configure DataHub's `DataHubValidationAction` for GX.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/integrations/integration_datahub.md:17]:
```suggestion
for more information on setting up and configuring `DataHubValidationAction` for GX.
```

File: docs/docusaurus/docs/integrations/integration_zenml.md [hunk 14]
```diff
@@ -8,334 +8,14 @@ sidebar_label: "ZenML"
 sidebar_custom_props: { icon: 'img/integrations/zenml_icon.png' }
 ---
 
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+ZenML is an extensible, open-source framework for creating production-ready, portable machine learning (ML) pipelines.  Integrating Great Expectations (GX) with ZenML helps data scientists and ML engineers to make data profiling and validation an integral part of their
```
Comment (kwcanuck) [inline docs/docusaurus/docs/integrations/integration_zenml.md:11]:
```suggestion
ZenML is an extensible, open-source framework for creating production-ready, portable machine learning (ML) pipelines.  Integrating Great Expectations (GX) with ZenML can help make data profiling and validation an integral part of your production ML toolset and workflows.
```

File: docs/docusaurus/docs/integrations/integration_zenml.md [hunk 15]
```diff
@@ -8,334 +8,14 @@ sidebar_label: "ZenML"
...
+production ML toolset and workflows.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/integrations/integration_zenml.md:12]:
```suggestion

```

**Review by kwcanuck — APPROVED** [review]

**Review by donaldheppner — APPROVED** [review]

**Review by joshzzheng — COMMENTED** [review]
- should athena still be here? it's still using block config
- mistake in prerequisite for prefect tutorial
- note: once ydata is live, we need to link it to that
- i think the sentence in info is unclear. instead of "The most recent version of GX supported by this guide is GX 0.15.50.", we should say "The most recent version of GX that supports this integration is GX 0.15.50".
- some preqs say "Python 3", shouldn't it be >Python 3.10(?)
- why do we say "Doing it?": https://deploy-preview-8173--niobium-lead-7998.netlify.app/docs/deployment_patterns/how_to_use_great_expectations_with_prefect/#doing-it

**Pull request merged**